pennylane/templates/broadcast.py

```diff
@@ -82,41 +105,16 @@ def _preprocess(parameters, pattern, wires):
         wire_sequence, parameters: preprocessed pattern and parameters
     """
 
-    custom_pattern = None
-
     if isinstance(pattern, str):
+        _wires = wires
```

Comment:
What's the reason for making this 'private'?

Reply:
I couldn't come up with a better name lol. I'm open to suggestions.